internal/plugin/service/organization/organization_application_user_token.go

```diff
@@ -108,24 +108,25 @@ func (r *organizationApplicationUserToken) Schema(
 	resp *resource.SchemaResponse,
 ) {
 	resp.Schema = util.GeneralizeSchema(ctx, schema.Schema{
-		Description: util.BetaDescription("Creates and manages an organization application user token in Aiven."),
+		Description: util.LimitedAvailabilityDescription("Creates and manages an authentication token for an application user in Aiven." +
+			" Review the [best practices](https://aiven.io/docs/platform/concepts/application-users#security-best-practices) for securing application users and their tokens."),
 		Attributes: map[string]schema.Attribute{
 			"id": schema.StringAttribute{
-				Description: "Compound identifier of the organization application user token.",
+				Description: "Compound identifier of the application user's token.",
 				Computed:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
 			"organization_id": schema.StringAttribute{
-				Description: "Identifier of the organization the application user token belongs to.",
+				Description: "The ID of the organization the application user belongs to.",
 				Required:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"user_id": schema.StringAttribute{
-				Description: "Identifier of the application user the token belongs to.",
+				Description: "The ID of the application user the token belongs to.",
```

Comment:
I think we should word it like: "The ID of the application user the token should belong to.", because the token here wasn't created yet at this stage, and we are creating the token, but it doesn't belongs to it before it was created.